<launch>

    <include file="$(find rplidar_ros)/launch/s2.launch">
    </include>

    <node pkg="charlie" type="tf_broadcaster.py" name="tf_broadcaster" output="screen"/>
    
    <!-- 로봇 URDF 모델 -->
    <param name="robot_description" command="xacro $(find charlie)/urdf/test.urdf.xacro"/>

    <!-- robot_state_publisher-->
    <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" output="screen">
        <param name="publish_frequency" value="50"/>
    </node>

    <!-- Hector Mapping-->
    <include file="$(find hector_mapping)/launch/mapping_default.launch">
    </include>

    <!-- RViz -->
    <node name="rviz" pkg="rviz" type="rviz" args="-d $(find charlie)/rviz/slam.rviz" required="true" />
</launch>

